test/functions/loss/softmax_cross_entropy_test.rb

```diff
@@ -237,23 +212,17 @@ def check_backward(x_data, t_data, g_data, class_weight)
     Chainer::check_backward(func, [x_data, t_data], g_data, eps: 0.02,  dtype: @check_backward_options_dtype)
   end
 
-  data(data)
-  def test_backward(data)
-    _setup(data)
+  def test_backward()
     check_backward(@x, @t, @g, @class_weight)
   end
 end
 
 class TestSoftmaxCrossEntropyInvalidReduce < Test::Unit::TestCase
-  use_cudnn    = ['always', 'auto', 'never'],
-  normalize    = [true, false]
-  cache_score  = [true, false]
+  data(:use_cudnn,   ['always', 'auto', 'never'], group: :reduce, keep: true)
+  data(:normalize,   [true, false],               group: :reduce, keep: true)
+  data(:cache_score, [true, false],               group: :reduce, keep: true)
```

Comment:
We can drop `group:` when we only use one data matrix group.

Reply:
Thank you for pointing out.
I fixed it.